APCAppCore/APCAppCore/UI/TabBarControllers/Activities/APCActivitiesViewSection.m

```diff
@@ -198,7 +198,7 @@ - (void) reduceToIncompleteTasksOnTheirLastLegalDay
                                                                                                  BOOL __unused *stopIterating) {
         BOOL keepThisOne = NO;
         
-        if (! taskGroup.isFullyCompleted && taskGroup.expiresToday)
+        if (! taskGroup.isFullyCompleted && [taskGroup expiresOnOrBeforeDate:self.today])
```

Comment:
Its filtered before we get there. Should we change the method name to better explain what's going on?